ui/apps/platform/src/utils/queryStringUtils.ts

```diff
@@ -4,8 +4,8 @@ export type BasePageAction = 'create' | 'edit';
 
 export type ExtendedPageAction = BasePageAction | 'clone' | 'generate';
 
-export function getQueryObject<T>(search: string): T {
-    return qs.parse(search, { ignoreQueryPrefix: true }) as unknown as T;
+export function getQueryObject<T extends qs.ParsedQs>(search: string): T {
```

Comment:
The only usages of this function supply a generic type that is a subtype of `qs.ParsedQs` so specifying the constraint here lets us get rid of the unsafe `as unknown as T` cast. There wasn't any code that relied on this cast, but if it did it would likely fail at runtime due to the return type of `qs.parse`.